### PR TITLE
New NodeJS version

### PR DESCRIPTION
**Pull request by NicholasPedroso**
## Description
In few word, describe what’s your Pull Request about.

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/resources/changelog/current-year.pt.md [hunk 1]
```diff
@@ -6,6 +6,24 @@ Descubra tudo sobre as novas versões e atualizações das integrações do Merc
 
 ## Dezembro 2020
 
+### 7 de novembro
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.pt.md:9]:
```suggestion
### 7 de dezembro
```

File: guides/resources/changelog/current-year.en.md [hunk 2]
```diff
@@ -6,6 +6,28 @@ Find out everything about the new versions and updates of Mercado Pago integrati
...
+>
+> New version of SDK NodeJS
+>
+> NEW_VERSION: NEW VERSION
+>
+> PRODUCT: SDK NODEJS
+
+We have released a new version of SDK NodeJS 1.5.2.
+
+- Add `statement_descriptor` option to Preferences
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.en.md:21]:
```suggestion
```

File: guides/resources/changelog/current-year.en.md [hunk 3]
```diff
@@ -6,6 +6,28 @@ Find out everything about the new versions and updates of Mercado Pago integrati
...
+
+- Credentials now can be set dynamically
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.en.md:23]:
```suggestion
```

File: guides/resources/changelog/current-year.en.md [hunk 4]
```diff
@@ -6,6 +6,28 @@ Find out everything about the new versions and updates of Mercado Pago integrati
...
+> New version of SDK NodeJS
+>
+> NEW_VERSION: NEW VERSION
+>
+> PRODUCT: SDK NODEJS
+
+We have released a new version of SDK NodeJS 1.5.2.
+
+- Add `statement_descriptor` option to Preferences
+
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.en.md:22]:
```suggestion
```

File: guides/resources/changelog/current-year.en.md [hunk 5]
```diff
@@ -6,6 +6,28 @@ Find out everything about the new versions and updates of Mercado Pago integrati
...
+- Credentials now can be set dynamically
+
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.en.md:24]:
```suggestion
```

File: guides/resources/changelog/current-year.en.md (hunk 3 above)
Comment (pamalv7) [reply guides/resources/changelog/current-year.en.md:23]: No está en los otros idiomas.

File: guides/resources/changelog/current-year.en.md (hunk 2 above)
Comment (pamalv7) [reply guides/resources/changelog/current-year.en.md:21]: No está en los otros idiomas.

**Review by pamalv7 — CHANGES_REQUESTED** [review]

**Review by pamalv7 — APPROVED** [review]

**Review by elvisjhonataoliveira — APPROVED** [review]

**Pull request merged**